jbmc/src/java_bytecode/README.md

```diff
@@ -25,7 +25,80 @@ To be documented.
 
 \subsection java-class-section How a java program / class is represented in a .class
 
-To be documented.
+Every Java class is compiled into a .class file, inner classes, anonymous
+classes or classes for tableswitches are compiled into their own .class
+files.
+
+There exists an [official
+specification](https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html)
+
+Each class files contains information about its version, the constant pool,
+information about the contained class, its super-class, its implemented
+interfaces, its fields, methods and finally additional attributes.
```

Comment:
maybe append "... such as exceptions thrown by a method or inner classes the class contains."